cg/services/order_validation_service/workflows/fastq/models/sample.py

```diff
@@ -0,0 +1,14 @@
+from cg.models.orders.sample_base import PriorityEnum, SexEnum
+from cg.services.order_validation_service.constants import ElutionBuffer
+from cg.services.order_validation_service.models.sample import Sample
+
+
+class FastqSample(Sample):
+    concentration_ng_ul: float | None = None
+    elution_buffer: ElutionBuffer
```

Comment:
@islean ? Atlas says it is optional
```suggestion
    elution_buffer: ElutionBuffer | None = None
```

Reply:
I see no way in which you can submit an order without the buffer set in this order type so I think it should be considered required: 
<img width="1183" alt="image" src="https://github.com/user-attachments/assets/a4b40f53-9729-4f82-b4b3-bd5c6274b2d7">

Atlas will have some confusing statements, since our different softwares are not always in agreement. For fastq orders, I interpret the Atlas page as saying "this is required in the FE but the BE gives it a default value as well". Simply having it as required in the BE makes more sense to me